packages/manager/apps/dedicated/client/app/components/dedicated-cloud/user/rights/edit/translations/Messages_fr_FR.json

```
@@ -34,9 +34,9 @@
   "dedicatedCloud_USER_right_vmNetwork_noAccess": "Aucun accès",
   "dedicatedCloud_USER_right_vmNetwork_READONLY": "Accès en lecture seule",
   "dedicatedCloud_USER_right_vmNetwork_readonly": "Accès en lecture seule",
-  "dedicatedCloud_USER_right_vlan": "Accès au V(x)Lans",
-  "dedicatedCloud_USER_right_vlan_help_EPCC": "Droits de management sur la partie réseau privé (VxLan pour les Dedicated Cloud et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
-  "dedicatedCloud_USER_right_vlan_help_MBM": "Droits de management sur la partie réseau privé (VxLan pour les Essentials et VLAN pour les SDDC et les packs Hosted Private Cloud Premier)",
+  "dedicatedCloud_USER_right_vlan_EPCC": "Accès au V(x)Lans",
+  "dedicatedCloud_USER_right_vlan_MBM": "Accès au VLans",
```

```suggestion
  "dedicatedCloud_USER_right_vlan_MBM": "Accès au VLan",
```
(same for: https://github.com/ovh/manager/pull/4041/files#diff-443ebe31bc9b786d3fd0ce07fb7383fc0a6deaad4fb729ab244801ddb14efce8R41 or all occurences of `au` + `VLan`)

---

Thanks a lot @JeremyDec . The sentence has been corrected as `Accès aux VLANs`